R/getRequiredColumns.R

```diff
@@ -18,11 +19,18 @@ getRequiredColumns<-function(standard,chart="eDish"){
     tolower(chart)=="edish"
   )
   
+  metadata <- safetyGraphics::getSettingsMetadata(
+    charts = chart, 
+    cols=c("setting_required","adam","sdtm")
+  )
+  
```

Comment:
Should be able to use use `filter_expr=setting_required==TRUE` instead of the filter below.

Reply:
good call